proxy/src/control/discovery.rs

```diff
@@ -327,8 +333,12 @@ where
 
         while let Some(auth) = self.reconnects.pop_front() {
             if let Some(set) = self.destinations.get_mut(&auth) {
-                set.query = DestinationServiceQuery::connect(client, &auth, "reconnect");
-                return true;
+                set.query = DestinationServiceQuery::connect_maybe(
+                    &self.default_destination_namespace,
+                    client,
+                    &auth,
+                    "reconnect");
+                return true; // TODO: continue?
```

Comment:
What's this comment referring to? Perhaps this should either be done, or the comment expanded explaining more.

Reply:
> What's this comment referring to? Perhaps this should either be done, or the comment expanded explaining more.

I removed the comment. It was a note to self, not intended to be in the PR.